src/main/java/it/gov/pagopa/payment/service/QRCodePaymentServiceImpl.java

```diff
@@ -17,16 +23,22 @@ public class QRCodePaymentServiceImpl implements QRCodePaymentService {
   private final QRCodePreAuthService qrCodePreAuthService;
   private final QRCodeAuthPaymentService qrCodeAuthPaymentService;
   private final QRCodeConfirmationService qrCodeConfirmationService;
+  private final TransactionInProgressRepository transactionInProgressRepository;
+  private final TransactionInProgress2SyncTrxStatus transactionMapper;
```

Comment:
I found transactionMapper too generic for a Facade class
```suggestion
  private final TransactionInProgress2SyncTrxStatus transaction2statusMapper;
```